### DISCUSSION TITLE
docker-compose build Error

### DISCUSSION THREAD

**Discussion by mickymkumar** [magma/magma#7291] [Q&A]
~/magma/feg/gateway/docker$ sudo docker-compose build
WARNING: The USE_GY_FOR_AUTH_ONLY variable is not set. Defaulting to a blank string.
WARNING: The GY_SUPPORTED_VENDOR_IDS variable is not set. Defaulting to a blank string.
WARNING: The GY_SERVICE_CONTEXT_ID variable is not set. Defaulting to a blank string.
ERROR: The Compose file './docker-compose.yml' is invalid because:
services.aaa_server.depends_on contains an invalid type, it should be an array
services.control_proxy.depends_on contains an invalid type, it should be an array
services.csfb.depends_on contains an invalid type, it should be an array
services.eap_aka.depends_on contains an invalid type, it should be an array
services.eap_sim.depends_on contains an invalid type, it should be an array
services.eventd.depends_on contains an invalid type, it should be an array
services.feg_hello.depends_on contains an invalid type, it should be an array
services.health.depends_on contains an invalid type, it should be an array
services.magmad.depends_on contains an invalid type, it should be an array
services.radiusd.depends_on contains an invalid type, it should be an array
services.redis.depends_on contains an invalid type, it should be an array
services.s6a_proxy.depends_on contains an invalid type, it should be an array
services.s8_proxy.depends_on contains an invalid type, it should be an array
services.session_proxy.depends_on contains an invalid type, it should be an array
services.swx_proxy.depends_on contains an invalid type, it should be an array

**Comment by karthiksubraveti** [comment]
This sounds like a typo in the yml file. Have you made any local changes ? 
@uri200 Have you seen this earlier ?
